index.html

```diff
@@ -0,0 +1,347 @@
+            <div class="user__info">
+                <div class="name__period">
+                    <p class="name">Roshan Jossey</p>
+                    <p class="period">4 years ago</p>
+                </div>
+                <p class="description">
+                    I've started on a project to help beginners to get started on contributing to open source projects. It's a hands-on tutorial to learn the general workflow in github. Check it out at <a href="#">https://github.com/Roshanjossey/first-contributions</a>
+                </p>
+                <div class="likes">
+                    <span><i class="far fa-thumbs-up"></i>8</span>
+                    <span><i class="far fa-thumbs-down"></i></span>
+                    <span>REPLY</span>
+                </div>
+            </div>
+        </div>
+        <!-- <div class="comment__box">
+            <div class="user__id"><img src="https://yt3.ggpht.com/ytc/AKedOLR9zLmLnLiahOXDtMiIBo7bQ-XpnQCU0zZt5mV2fw=s48-c-k-c0x00ffffff-no-rj" alt=""></div>
+            <div class="user__info">
+                <div class="name__period">
+                    <p>Roshan Jossey</p>
+                    <p>4 years ago</p>
+                </div>
+                <p class="description">
+                    I've started on a project to help beginners to get started on contributing to open source projects. It's a hands-on tutorial to learn the general workflow in github. Check it out at <a href="#">https://github.com/Roshanjossey/first-contributions</a>
+                </p>
+                <div class="likes">
+                    <span><i class="far fa-thumbs-up"></i>8</span>
+                    <span><i class="far fa-thumbs-down"></i></span>
+                    <span>REPLY</span>
+                </div>
+            </div>
+        </div> -->
+        <div class="comment__box">
+            <div class="user__id"><img src="https://yt3.ggpht.com/ytc/AKedOLRBVfmEBA5woW8bisUB5rh-g7GEvNwsLZkA1ZI0Ig=s48-c-k-c0x00ffffff-no-rj" alt=""></div>
+            <div class="user__info">
+                <div class="name__period">
+                    <p class="name">christine Fredricksen</p>
+                    <p class="period">1 year ago</p>
+                </div>
+                <p class="description">
+                    Thanks so much ! This video was really helpful and to the point.
+                </p>
+                <div class="likes">
+                    <span><i class="far fa-thumbs-up"></i></span>
+                    <span><i class="far fa-thumbs-down"></i></span>
+                    <span>REPLY</span>
+                </div>
+            </div>
+        </div>
+        <div class="comment__box">
+            <div class="user__id"><img src="https://yt3.ggpht.com/ytc/AKedOLTtm3xTw6YgY_vAvaR6Je41py9usCuyxlUW2D-MJw=s48-c-k-c0x00ffffff-no-rj" alt=""></div>
```

Comment:
When placing the img tag always ensure to fill the alt. Doing that improves your SEO ranking and is good for screen readers

Reply:
- [ ] Good describing the "Why" in your suggestion. Nice jobe 👍